core/Db/Schema/Mysql.php

```diff
@@ -102,6 +102,14 @@ public function getTablesCreateSql()
                             ) ENGINE=$engine DEFAULT CHARSET=utf8
             ",
 
+            'site_setting'    => "CREATE TABLE {$prefixTables}site_setting (
```

Comment:
We will store property settings in this table. The `setting_value` will be serialized as a `Setting` can define a type and we do not want to lose that type, eg an array.